<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winMenu</class>
 <widget class="QMainWindow" name="winMenu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu Général</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnClients">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStock">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inventaire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStaff">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Staff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContracts">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contrats</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblVersion">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
